README.md

```diff
@@ -8,4 +8,47 @@ In order to prevent sensitive credentials to be leaked, please follow this instr
 - Install [git-secrets](https://github.com/awslabs/git-secrets) in your computer.
 - Go into the repo root folder.
 - Run `git secrets --register-aws`.
-- Run `git config --local core.hooksPath .githooks/`.
\ No newline at end of file
+- Run `git config --local core.hooksPath .githooks/`.
+
+## Running tests
+
+The easiest way to run the tests locally is to run the test DB in docker.
+
+Ensure you have the following environment variables set on the terminal session you want to run your tests from:
+```
+POSTGRES_PORT=5433
+API_KEY='anything'
+RUNNING_TEST=true
+```
+
+Then, in the HRM root, run
+
+```shell
+docker compose -f ./docker-compose-test.yml up -d
+jest --verbose                                      # tests will run here 
+docker compose -f ./docker-compose-test.yml down    # if you forget this, the DB will not be cleared down & be in a dirty state for the next run, which could make tests unstable
```

Comment:
I think we can make it an npm command so the user never forgets to clean up the DB.
We could change our current test script from `package.json` to be:

```
"test:setup": "docker compose -f ./docker-compose-test.yml up -d",
"test:main": "POSTGRES_PORT=5433 API_KEY='anything' RUNNING_TESTS=true jest --passWithNoTests --forceExit --verbose",
"test:teardown": "docker compose -f ./docker-compose-test.yml down",
"test": "npm run test:setup && npm run test:main || npm run test:teardown",
```

But this doesn't solve the scenario where the user stops the process while running the tests (`ctrl+c`). In this case, he/she would have to call `npm run test:teardown` explicitly.

Reply:
We can do that sure - the npm task isn't Windows compatible so I don't use it myself which is why I didn't think to change it. (I suppose I can just run it in WSL or git bash though - probably git bash - running docker from WSL can get hairy since docker is ALSO running in WSL...)

The only way to get the tests to always run the teardown would be to run the tests in themselves in a container as part of the same `docker compose` as the DB, which is doable, just have to figure out how best to pass jest args down from the host to the hjest runner in the compose file, so you can run single tests or set verbose output or whatever